src/main/java/teammates/ui/pagedata/StudentFeedbackResultsPageData.java

```diff
@@ -235,4 +240,14 @@ private FeedbackResultsResponseTable createResponseTable(FeedbackQuestionAttribu
         }
         return responsesForRecipient;
     }
+
+    /**
+     * Remove the identifier from anonymous participant name or email, if any.
+     * @param identifier  The given name or email
+     * @return The name or email without any anonymous identifier
+     */
+    private String getDisplayableIdentifier(String identifier) {
```

Comment:
Oops, I only just realised the approach you were taking (temporarily find and replace the hash). It's okay for the method name to reference the hash then (maybe use `removeAnonymousHash`?) since the method will be removed when the hash is completely removed. But do add a `@deprecated` tag.